posts/exercism-io-is-now-exercism-org.md

```diff
@@ -0,0 +1,25 @@
+# Exercism has moved to exercism.org
+
+Exercism has a new home at exercism.org!
+
+For the last 8 years Exercism has lived at exercism.io and we decided it was time for a change!
+
+If you're interested, here are a few of the reasons we made the decision.
+
+Primarily, we feel that the `.org` domain better represents what Exercism is. 
+Most startups begin with finding whatever domain is available, and `.io` was a nice fit as it's often associated with I/O, but as we've grown and evolved, and become a larger platform, we've felt for a while that it was time to change to a more "official" seeming domain.
+We own `exercism.com`, `exercism.net`, and a few others, but we feel that `exercism.org` best represents our not-for-profit mission and gives the best first impression.
+As we also look to raise donations from both our community and from organisations, we felt that the more professional `.org` also offers more trust, especially as our name can sometimes give a slightly jarring first impression…
+
+Secondly, being on an `io` domain gives a little less safety than using one of the original TLDs. 
+Back in 2017 the [io authoritative nameservers were compromised](https://thehackerblog.com/the-io-error-taking-control-of-all-io-domains-with-a-targeted-registration/) using a pretty trivial method, and while mistakes happen and things were quickly rectified, this isn't the sort of thing that's going to happen to `.org` domains.
+We've also had sporadic issues with DNS resolving, which people being unable to access the site. 
+We've never been able to track this exactly down, but our best understanding is that something occasionally goes wrong at the `.io` level, rather than the AWS level.
```

Comment:
```suggestion
We've never been able to track this down exactly, but our best understanding is that something occasionally goes wrong at the `.io` level, rather than the AWS level.
```